carapax-i18n/src/lib.rs

```diff
@@ -0,0 +1,148 @@
+//! An i18n handler for carapax
+
+#![warn(missing_docs)]
+
+#[cfg(feature = "gettext")]
+mod gettext;
+#[cfg(feature = "gettext")]
+pub use gettext::*;
+
+#[cfg(feature = "fluent")]
+#[macro_use]
+extern crate rental;
```

Comment:
Can we use `rental::%macro%`?

Reply:
unfortunately `rental` crate doesn't support it :(